src/routes/about/privacy.svelte

```diff
@@ -0,0 +1,51 @@
+<script>
```

Comment:
#19 added a privacy policy with privacy options, but the actual policy content should be revised by the team